packages/newspringchurchapp/src/prayer/PrayerCard/PrayerCardConnected.tests.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { renderWithApolloData } from 'newspringchurchapp/src/utils/testUtils';
+
+import Providers from 'newspringchurchapp/src/Providers';
+import flagPrayerRequest from './flagPrayerRequest';
+import PrayerCardConnected from './PrayerCardConnected';
+
+const prayerId = [
+  {
+    id: 'PrayerRequest:b99bbd5002a0bf8d4c73e86afe9d4789',
```

Comment:
Is this an Id of an actual prayer in Rock? If so, what would happen if it gets deleted?

Reply:
@richarddubay I think this is fine because this is the dummy data response that we're providing to the mocks.